.travis.yml

```diff
@@ -29,8 +31,11 @@ python:
 matrix:
     fast_finish: true
 
+before_install:
+  - export CXX="g++-4.9" CC="gcc-4.9"
 
 install:
+  - gcc --version
```

Comment:
this is possibly misleading, because it will still still use the `gcc` on `PATH`, which is most likely different than the one picked by cmake (which takes into account the `CXX` variable)
In any case, cmake prints out at the beginning what compiler it's using